### DISCUSSION TITLE
Plugin directory

### DISCUSSION THREAD

**Discussion by treeandcoffee** [GetPublii/Publii#1865] [General] [answered]
Is there already a directory to third party plugins for publii?

**Comment by candidexmedia** [comment, marked as answer]
No, but I have an informal list of the ones I've come across here (including some from the old forum): https://github.com/GetPublii/Publii/discussions/1716#discussioncomment-10781651

Just missing a couple of new ones that have been posted in this forum.

**Reply by treeandcoffee** [reply]
Thank you very much for pointing me to your gists.
